Helper/Cart.php

```diff
@@ -2318,8 +2319,19 @@ public function collectDiscounts(
 
                         break;
                 }
+                $amount -= $roundedAmount;
                 $totalAmount -= $roundedAmount;
             }
+            if ($this->deciderHelper->isAPIDrivenIntegrationEnabled()
+                && $this->deciderHelper->isSkipCartDiscountTotalMismatch()
+                && $amount) {
+                usort($discounts, function ($a, $b) {
```

Comment:
could we not change discount orders here?
It would be nice to have initial order.
It also affects on the order of discounts displaying in bolt modal

Reply:
@vitaliyreznikov  I have removed the usort and add difference value to the amount of first element by default, actually we do not know which sale rule causes the difference, so it does not matter.

Reply:
In Andrew Christian case, the first discount can be $0.10 and the second $100
It's more logical to change the biggest discount

Reply:
Done